### DISCUSSION TITLE
How to leave a comment in the linked discussion group after sending a channel message?

### DISCUSSION THREAD

**Discussion by hexsix** [python-telegram-bot/python-telegram-bot#3199] [Q&A] [answered]
I want to achieve this goal:

1. My bot post a message in the channel. And the message is auto forwading to the linked discussion group.
2. **THE TASK**: I want my bot to post comments (replies) to the message in the discussion group. Finally in the channel we can see that the message has comments.

I know message_id is needed for reply, but I only got channel message_id in step 1. I found that the channel message_id and the discussion group message_id are different. Any ideas there?

**Comment by Poolitzer** [comment, marked as answer]
you add the bot to the dicussion group, it gets an update when the channel post happens in that group, reply to it.

**Reply by hexsix** [reply]
Thank you for your reply. I'll give it a try.

**Reply by hexsix** [reply]
It works, thank you very much.

**Reply by obitoscall** [reply]
Was
